### ISSUE TITLE
Amazon - Choose a Shipping Address

### ISSUE THREAD

**Issue by jfcarr** [ClearURLs/Addon#267]
Version 1.25.0, installed in Firefox, will not allow me to choose a new shipping address in the Amazon shopping cart.  When I select an address and click "Use this address", I am redirected to a page that says "Oops! We're sorry. The Amazon.com page you tried to access can be viewed only during the checkout process."  If I disable ClearURLs, this process works.

**Comment by miangraham** [comment]
I also started seeing this sometime in the past week or so (Firefox on amazon.co.jp) and disabled the extension as a temporary workaround. My extension version is 1.25.0 from July 28, 2022 so it must be a new change on Amazon's side.

After choosing a shipping address they bounce you to a url like https://www.amazon.*/gp/buy/payselect/handlers/display.html?hasWorkingJavascript=1, and I believe it's the hasWorkingJavascript parameter (or possibly some other intermediate query param I'm not seeing--they love redirects) getting stripped out that causes the breakage.

Not too familiar with the extension but I'm guessing the Amazon rules need a new excluded field to reflect the change on their end.

**Comment by tejasraykar01** [comment]
Yes, I'm dealing with this issue on Amazon as well. Disabling the extension seems to be the only workaround for now.

**Comment by DanCrank** [comment]
More specifically, turning off "filter" in the ClearURLs settings (leaving "badges" and "statistics" enabled) worked around the problem for me.
